tests/integ/test_training_compiler.py

```diff
@@ -150,7 +150,7 @@ def test_pytorch(
     with timeout(minutes=TRAINING_DEFAULT_TIMEOUT_MINUTES):
 
         hf = PyTorch(
-            py_version="py38",
+            py_version="py39",
```

Comment:
Please reuse the fixture from above to reduce manual effort here.

Reply:
Here it just tested the latest release version and it cannot use the fixture from above. I think we don't need to have fixture here because it only have one version(which should be the latest dlc's py_version.)

Reply:
Add a ToDo in conftest to create a fixture to get the latest py version.